src/Resources/views/bundles/SyliusAdminBundle/Product/_slugField.html.twig

```diff
@@ -0,0 +1,18 @@
+<div class="{% if slugField.vars.required %}required {% endif %}field{% if slugField.vars.errors|length > 0 %} error{% endif %} ui loadable form">
```

Comment:
This file shouldn't be here I guess.
Looks like same file exists at right place at `tests/Application/templates/...`

Reply:
Yeah, it is normal. The file under `src/Resources/views/bundles/SyliusAdminBundle/Product/_slugField.html.twig` is for copying it for users that will install it.
And the one under `tests/Application/...` is the copy I made for testing.

Isn't it that way ?

Reply:
As on option, you can try to require it from `tests/Application/templates/...` rather than copy.
I believe it will be painful to support both files in same condition if some changes will be required in future.

Reply:
So you would advice to keep the one in `tests/Application` and remove the one under `src/Resources`, and write in README to copy templates from `tests/Application` ?

Reply:
Yes :)

Reply:
@Roshyo , this also an option (which we using usually)